lib/webauthn/metadata/client.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require "jwt"
+require "net/http"
+
+module WebAuthn
+  module Metadata
+    class Client
```

Comment:
For the record: https://github.com/cedarcode/webauthn-ruby/pull/208#issuecomment-551920674.